src/ADAL.Common/AcquireTokenInteractiveHandler.cs

```diff
@@ -78,7 +80,15 @@ public AcquireTokenInteractiveHandler(Authenticator authenticator, TokenCache to
             this.DisplayableId = userId.DisplayableId;
             this.UserIdentifierType = userId.Type;
 
-            this.LoadFromCache = (tokenCache != null && this.promptBehavior != PromptBehavior.Always && this.promptBehavior != PromptBehavior.RefreshSession);
+            if (!string.IsNullOrEmpty(claims))
```

Comment:
Why only a check for `IsNullOrEmpty` here whereas in other places you check for `IsNullOrWhiteSpace`?